### PR TITLE
Update Some dependencies

### PR DESCRIPTION
**Pull request by milleruntime**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -700,17 +700,17 @@
       <dependency>
         <groupId>org.hibernate</groupId>
         <artifactId>hibernate-validator</artifactId>
-        <version>5.1.3.Final</version>
+        <version>6.1.0.Alpha4</version>
```
Comment (ctubbsii) [inline pom.xml]: Probably shouldn't update to an alpha.

File: pom.xml [hunk 2]
```diff
@@ -1041,7 +1041,7 @@
               </ignoredUsedUndeclaredDependencies>
               <ignoredUnusedDeclaredDependencies>
                 <!-- auto-service isn't detected as use since the annotation has retention of source -->
-                <unusedDeclaredDependency>com.google.auto.service:auto-service:jar:1.0-rc4</unusedDeclaredDependency>
+                <unusedDeclaredDependency>com.google.auto.service:auto-service:jar:1.0-rc5</unusedDeclaredDependency>
```
Comment (ctubbsii) [inline pom.xml]: Can we put a wildcard here?

File: pom.xml [hunk 3]
```diff
@@ -243,12 +243,12 @@
...
         <groupId>commons-codec</groupId>
         <artifactId>commons-codec</artifactId>
-        <version>1.11</version>
+        <version>1.12</version>
       </dependency>
       <dependency>
         <groupId>commons-collections</groupId>
         <artifactId>commons-collections</artifactId>
-        <version>3.2.2</version>
+        <version>20040616</version>
```
[the remark this answers is not in the document]
Comment (milleruntime) [reply pom.xml]: Haha yeah, I will revert.

[the remark this answers is not in the document]
Comment (milleruntime) [reply pom.xml]: Do we want to go to 4?

[the remark this answers is not in the document]
Comment (ctubbsii) [reply pom.xml]: If we're able to, probably. Need to ensure it works, without conflicting with Hadoop, and other deps.

[the remark this answers is not in the document]
Comment (milleruntime) [reply pom.xml]: This is a bigger change, going to open a ticket.

[the remark this answers is not in the document]
Comment (milleruntime) [reply pom.xml]: https://github.com/apache/accumulo/issues/1093

[the remark this answers is not in the document]
Comment (ctubbsii) [reply pom.xml]: Should at least revert to 3.2.2 in the interim.

**Comment by milleruntime** [comment]
I am going to do update EasyMock, Powermock and Junit in another PR.

File: pom.xml [hunk 4]
```diff
@@ -1041,7 +1041,7 @@
               </ignoredUsedUndeclaredDependencies>
               <ignoredUnusedDeclaredDependencies>
                 <!-- auto-service isn't detected as use since the annotation has retention of source -->
-                <unusedDeclaredDependency>com.google.auto.service:auto-service:jar:1.0-rc4</unusedDeclaredDependency>
+                <unusedDeclaredDependency>com.google.auto.service:auto-service:jar:1.0-rc*</unusedDeclaredDependency>
```
Comment (ctubbsii) [inline pom.xml]: I actually meant a wildcard for the entire version, so we can change to whatever version comes later.

Comment (milleruntime) [reply pom.xml]: Whoops!  Fixed in 67cb5999dff16e40430ff6d1dfa3bbcec11f25e

**Pull request merged**